assets/components/chat.html

```diff
@@ -20,10 +20,20 @@ <h2 class="ui dividing header">Chat
           <div am-time-ago="{{$ctrl.getTime(chat)}}" class="date"></div>
         </div>
         <div class="extra text chat {{chat.type === 'user' ? '' : 'content minimal'}}">
+          <i class="plus icon" ng-if="chat.type === 'addVideo'"></i>
```

Comment:
this is what i don't like about angular. i guess in the future this could be cleaned up to a <chat-icon> component